src/openApiCodeGenClients/__test__/operation.test.js

```diff
@@ -42,5 +47,6 @@ describe('createOpenAPIClient', () => {
         });
         expect(spy).toHaveBeenCalledTimes(1);
         expect(spy.mock.calls[0][1].headers['X-Request-Id']).toBe('request-id');
+        expect(spy.mock.calls[0][1].headers['X-Request-Tenant-Id']).toBe('client-tenant-id-123');
```

Comment:
Please also add TC for the scenario when user don't have tenant id

Reply:
good shout. Done